structs.h

```diff
@@ -1,46 +0,0 @@
-#ifndef STRUCTS_H
-#define STRUCTS_H
-
-typedef struct Date{
-    int i_year;
-    int i_month;
-    int i_day;
-}Date;
-
-typedef struct Recipe{
-    char c_Recipe_ID[37];
-    char c_Recipe_Name[51];
-    char c_Recipe_Description[5001];
-    struct Recipe * R_next_Recipe;
```

Comment:
Alternatively you can create a list for storing recipes, separately from the data storage structure.
struct RecipeList{
    struct Recipe * recipe;
    struct Recipe * next recipe;
}
It would be more transparent for the backend, and you will be able to change easily the storing structure.